plugins/BeamerPresentation/plugin.ini

```diff
@@ -0,0 +1,15 @@
+[about]
+## Author / Copyright notice
+author=Lukas Heindl
+
+description=Step through LaTeX beamer presentation. Typical workflow: First use the function which removes all pdf-pages related to an animation. Then, step through the slides with the specialized functions instead of manually going forward/backwards. The plugin will decide whether to in-/decrese the background page (for the animation) or go to the next slide.
```

Comment:
```suggestion
description=Step through LaTeX beamer presentation. Typical workflow: First use the function which removes all pdf-pages related to an animation. Then, step through the slides with the specialized functions instead of manually going forward/backwards. The plugin will decide whether to in-/decrease the background page (for the animation) or go to the next slide.
```